webpack/assets/javascripts/react_app/components/PasswordStrength/__tests__/PasswordStrengthActions.test.js

```diff
@@ -0,0 +1,9 @@
+import { updatePassword, updatePasswordConfirmation } from '../PasswordStrengthActions';
+
+describe('PasswordStrength actions', () => {
+  it('should update password', () =>
+    expect(updatePassword('some-password')).toMatchSnapshot());
```

Comment:
I like this simplicity, though with async actions, should we adopt redux [suggestion](https://redux.js.org/recipes/writing-tests) ?

Reply:
Yes in general, I will try to experience with it more in later PR and we will continue to discuss it there.